core/modules/mysql/MySqlConfig.h

```diff
@@ -101,6 +114,14 @@ class MySqlConfig {
     std::string dbName;
     size_t maxTableSizeMB;
 
+    /**
+     * Get the connection object to use if one was provided.
+     * This is useful for unit testing.
+     */
+    std::shared_ptr<sql::SqlConnection> getConnection() const;
+
+private:
+    std::shared_ptr<sql::SqlConnection> _sqlConnection;
```

Comment:
I tried once, it was going to require some refactoring that seemed heavy at the time. Maybe now (after the recent demolition and reconstruction) it will seem not too bad. I'll go take a look...

Reply:
(ok, fixed it)